### PR TITLE
Fix stable grading criteria on `Gameplay differences in osu!(lazer)`

### PR DESCRIPTION
**Pull request by felipemarins**
Fixed the stable grading criteria on the lazer differences article and matched it with the one on the [Grade](https://osu.ppy.sh/wiki/en/Gameplay/Grade) article, as that is seemingly more correct than the one on the former article.
I decided to do it after this convo on the osu! dev discord: https://discord.com/channels/188630481301012481/1097318920991559880/1304613420867522562

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

**Review by cl8n — COMMENTED** [review]
i dont think splitting osu and taiko is necessary. the rest looks good

**Comment by felipemarins** [comment]
I didn't know if I should place the "GREATs" or "300s" first, so I just tried to follow the ordering of the "MEHs/50s" that is on master. Also, I didn't put the "MEHs" back as I understand those as a reference to taiko judgements, so it wouldn't apply here.

File: wiki/Client/Release_stream/Lazer/Gameplay_differences_in_osu!(lazer)/en.md [hunk 1]
```diff
@@ -24,10 +24,10 @@ In stable, the accuracy (and judgement) requirements for each [grade](/wiki/Game
 | Grade | osu! / osu!taiko | osu!catch | osu!mania |
 | :-: | :-- | :-- | :-- |
 | SS | 100% | 100% | 100% |
-| S | ≥90% (≤1% MEHs/50s, no misses) | ≥98% | ≥95% |
-| A | ≥80% (no misses) or ≥90% | ≥94% | ≥90% |
-| B | ≥70% (no misses) or ≥80% | ≥90% | ≥80% |
-| C | ≥60% | ≥85% | ≥70% |
+| S | >90% GREATs/300s (≤1% 50s, no misses) | >98% | >95% |
```
Comment (peppy) [inline wiki/Client/Release_stream/Lazer/Gameplay_differences_in_osu!(lazer)/en.md:27]: why was the "MEH" term removed here while "GREAT" was left?

Comment (felipemarins) [reply wiki/Client/Release_stream/Lazer/Gameplay_differences_in_osu!(lazer)/en.md]: Fixed. I'm sorry for this. I forgot that MEH judgements don't exist on taiko and ended up misunderstanding the reasoning to include them here.

**Review by peppy — APPROVED** [review]
Seems fine, will leave it approved for someone to double-check.

**Review by Hiviexd — APPROVED** [review]

**Pull request merged**